sceptre/synapsedev/templates/accounts.yaml

```diff
@@ -71,6 +71,16 @@ Resources:
       LoginProfile:
         Password: !Ref InitNewUserPassword
         PasswordResetRequired: true
+  AWSIAMLukeMoenningUser:
```

Comment:
Could you rename as 'AWSIAMLukeMoenningSvc'. These IAM accounts are really service accounts to run your stack instances under.